tests/end_to_end/candid_rpc/class_syntax/async_await/benchmarks_async_await.json

```diff
@@ -0,0 +1,68 @@
+{
+    "current": {
+        "version": "0.25.0",
+        "benchmarks": [
+            [
+                { "__bigint__": "1728686479544102262" },
```

Comment:
Let's rethink the data structure now that the names are working

Reply:
yep lets do it!

Reply:
done